src/index.html

```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="url" method="post"
+          class="search-bar search-bar--big" data-qa="big"
+    >
+      <label for="big" class="search-bar__label search-bar__label--big">
+        <input type="text" id="big"
+               data-qa="keypress" class="search-bar__input search-bar__input--big"
+               placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
+
+    <form action="url" method="post"
```

Comment:
typically for method we use uppercase
```suggestion
    <form action="url" method="POST"
```

Reply:
oh, that's right, in that case, I removed the method attribute as it is automatically set to "GET"